src/services/MakeStreamZip.php

```diff
@@ -80,7 +80,14 @@ public function __destruct()
      */
     public function getFileName(): string
     {
-        return 'export.elabftw.zip';
+        $this->idArr = explode(' ', $this->idList);
```

Comment:
in fact this should be moved into the constructor and idList property removed, only idArr. Here you're making a code repetition.

Reply:
edit: done

Reply:
Ok, I will rebase my branch on the last commit on hypernext and delete this line.